components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.js

```diff
@@ -386,7 +386,11 @@ document.observe('xwiki:dom:loading', function() {
   }
 
   document.observe('xwiki:multisuggestpicker:selectionchanged', function() {
-    document.fire('xwiki:livetable:patients:filtersChanged');
+    if (XWiki.currentDocument.space == 'data') {
+      document.fire('xwiki:livetable:patients:filtersChanged');
+    } else {
+      document.fire('xwiki:livetable:families:filtersChanged');
```

Comment:
This isn't very extensible... Is there a way to get the table name from somewhere? Is this generic enough:

```
if ($$(".entity-directory")[0]) {
  document.fire('xwiki:livetable:' + $$(".entity-directory")[0].id.replace(/-.*/, '') + ':filtersChanged');
}
```